ui/src/timeMachine/actions/queries.ts

```diff
@@ -171,6 +168,10 @@ export const executeQueries = (abortController?: AbortController) => async (
     const duration = window.performance.now() - startTime
 
     let statuses = [[]] as StatusRow[][]
+    const {
+      alertBuilder: {id: checkID},
+    } = state
+
```

Comment:
Moved this down so it would be place a little closer to the source of where `checkID` is being used